src/test/groovy/edu/oregonstate/mist/beaverbus/ResourceMapperTest.groovy

```diff
@@ -0,0 +1,92 @@
+package edu.oregonstate.mist.beaverbus
+
+import edu.oregonstate.mist.api.jsonapi.ResourceObject
+import edu.oregonstate.mist.beaverbus.core.RouteAttributes
+import edu.oregonstate.mist.beaverbus.core.Stop
+import groovy.transform.CompileStatic
+import groovy.test.GroovyAssert
+import org.junit.Test
+
+import java.time.Instant
+
+@CompileStatic
+class ResourceMapperTest {
+    ResourceMapper mapper = new ResourceMapper()
+    BeaverBusUriBuilder uriBuilder = new BeaverBusUriBuilder(new URI("http://example.com/"))
+
+    @Test
+    void testMapRoute() {
+        def input = new RouteWithSchedule(
+                "Description": "West Route 2",
+                "EncodedPolyline": "q__oGjlmoV?a@?eC?i@?mA?Y??V@`@AV?HAD?FAJCHC~@Y??NGTKDAFADAH?LAhA?|@?j@?d@?H?@?B?D@F@NBJ?^?^???nC?LD???xA?`F?fABT?N??@FBRBRDRFTDR??HTDJHJFH???@HHf@d@HHJFHDJDJBNBL@B?V@N?FAFAHAFEJIFIHMfAeBb@m@DEDCFAPC??FYBS???CBS@U@O?Q?O?OAM?OAOCQAOAKAIAG??Z?`@Ad@?`@?D?B?@AB?BABAZS??\\~@XbAJb@Np@Hh@Df@Db@BPBb@@JB^@\\?v@AnCCnC??}@@i@AM???sB?aB?gBAwC?m@Ae@?w@?_AA}@?K?cC?A?I?C?KOCCAAECEAEAICG?O?i@?kAAeB?O???wE????gB?gA?gA?g@?m@?sF",
```

Comment:
Codenarc complaint here

Reply:
Codenarc is being needlessly prissy. I suggest disabling the line length rule in test files; what do you think?

Reply:
I don't know if I'm on board with that. Line length is meant to help with readability. The fact that its a test file has nothing to do with that - tests should be as readable and consistent as any other file. @tsoliangwu0130 thoughts?

Reply:
Nothing is going to make this line more readable -- it's a string of encoded garbage. The best thing we can do is get it out of the reader's way by letting it fall off the right side of the screen.

Reply:
Part of me agrees @jaredkosanovic 's thoughts, it is important to have these rules to keep the code clean and neat especially in a team repo. Without having this limit, the code is so easy to get too long. However, I also agree that sometimes Codenarc is really prissy. Currently, the limit of line length is [100](https://github.com/osu-mist/web-api-skeleton/blob/3019769d149ef251ac87a2b33621123ed6bb4f01/config/codenarc/codenarc.groovy#L77), I guess my suggestion will be to increase the limit to 120 and still keep Codenarc to check every files. (or maybe we can adjust the rule priority, but I am not sure how does that work yet.)

Reply:
@magical I agree that a long string of garbage is maybe an exception to the rule, but I don't think we should disable line length check for _all_ test files based on this use case.

Reply:
@tsoliangwu0130 
1. this line is far longer than 120 characters, so increasing the line length limit to 120 wouldn't help here
2. i think a line length of 100 is still reasonable in most cases

Reply:
@magical True. My main thinking is I still lean to separate the encoded string by separate lines, just like the RSA private key file (`id_rsa`), if set the limit to 120 characters at least we can have fewer lines (but I agree it doesn't really make big difference).